trojsten/contests/templates/trojsten/contests/dashboard.html

```diff
@@ -0,0 +1,31 @@
+{% extends "trojsten/layout/main.html" %}
+{% load statements_parts %}
+{% load archive_parts %}
+
+{% block title %}Nástenka{% endblock %}
+
+{% block page_header %}
+<h1>Nástenka</h1>
```

Comment:
Stringy by bolo fajn mat translatable.

Reply:
Rozmýšľal som nad tým, ale neviem, ako vážne berieme preklady, keďže v kóde je to tiež často všelijaké. Ale zmením